src/github.com/mozilla-services/pushgo/simplepush/app.go

```diff
@@ -264,15 +262,23 @@ func (a *Application) GetWorker(uaid string) (worker Worker, ok bool) {
 	return
 }
 
-func (a *Application) AddWorker(uaid string, worker Worker) {
+func (a *Application) AddWorker(uaid string, worker Worker) (added bool) {
```

Comment:
Nit: Some other return var name. Since the client will always be 'added to the map'. Maybe call it 'replaced' to indicate if the client replaced a prior entry?